### ISSUE TITLE
Edit Savenumber?

### ISSUE THREAD

**Issue by xTheSolutionNOT** [JulianOzelRose/TRR-SaveMaster#5]
Is it possible to edit the save number? For example, in TR 2 my saves are:

The Great Wall - 1
Venice -  6
Bartolis Hideout - 15

etc, and i'd like to change it to

The Great Wall - 1
Venice -  2
Bartolis Hideout - 3

It's not that important but it triggers my inner monk. So I wanted to ask if it's possible to edit that and if you could add it please?

**Comment by JulianOzelRose** [comment]
Try out this version:  [TRR-SaveMaster.zip](https://github.com/JulianOzelRose/TRR-SaveMaster/files/14488410/TRR-SaveMaster.zip)

Just don't set the save number to 0, or the editor won't recognize the savegame anymore. I have to a change a few more things for that to work. By the way, I added the ability to edit statistics. Both the latest version on this repo (v1.00) as well as the one attached can edit game statistics.

**Comment by xTheSolutionNOT** [comment]
I tested it across all three games and it works. I edited the save numbers for PS4 and Switch and it worked as intended. I filled all slots in all games to be sure and all were edited just fine and they seem to load all without errors. I didn't load into all saves, but I tested 5 slots for each game and they all worked. Statistics I edited for these slots I loaded into also worked 


Really awesome work once again.
